config/features/topic_page/views.view.taxonomy_term.yml

```diff
@@ -1,362 +0,0 @@
-uuid: 938a7046-1ec6-4f3d-910a-fb0a5655af80
```

Comment:
@quamsta Just checking in to see what the purpose of having this view was in the first place. I can see quite a few differences between the view that was created as part of this feature and the version in the default profile. Was it actually necessary to have this view overridden? If so, we should probably talk through that since it is potentially changing the view for every instance where term content is shown. Also, this will need to be implemented as a patched config override.

Reply:
That's correct, @joewhitsitt . Sorry it's been hectic over here, haven't been keeping up

Reply:
Maybe we can see if CS2 will allow a partial split for this instead of a complete split.

Reply:
Yeah, for some reason I thought I was only applying this config split option to sites that had the topic collection split enabled? The whole config split thing is often intensely confusing to me.

Reply:
It is only being applied to topic collection sites, but it is doing a complete fork of the view versus just changing the filter on the view. So if we update the default view, topic_collection doesn't automatically get those changes.

Reply:
Ohh okay. That makes sense. Willing to help try and help fix that, although I see that the view has been removed from the diff already. 

I believe, as @joewhitsitt said, this highlighted filter in the following screenshot was the only thing I really needed for the Topic Collection to be excluded from `/tags/[tag-name]` due to the way the Topic Collection handles its own relationship with tags.

<img width="533" alt="Screen Shot 2022-09-02 at 8 58 55 AM" src="https://user-images.githubusercontent.com/472923/188165499-472ce541-a167-4f2c-babf-7e35dc9b85de.png">

Reply:
i am about to push a change that moves the views touched by this feature to complete. As partial, i get both a patched version and a complete version. moving them to complete makes it so that we only get on version of the config and that no functionality is lost. I will document this on https://github.com/uiowa/uiowa/issues/5427